src/SFA.DAS.ApprenticePortal/Application/Homepage/Queries/GetApprenticeHomepageQueryHandler.cs

```diff
@@ -0,0 +1,41 @@
+﻿using MediatR;
+using SFA.DAS.ApprenticePortal.InnerApi.ApprenticeAccounts.Requests;
+using SFA.DAS.ApprenticePortal.InnerApi.ApprenticeCommitments.Requests;
+using SFA.DAS.ApprenticePortal.Models;
+using SFA.DAS.SharedOuterApi.Configuration;
+using SFA.DAS.SharedOuterApi.Interfaces;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.ApprenticePortal.Application.Homepage.Queries
+{
+    public class GetApprenticeHomepageQueryHandler : IRequestHandler<GetApprenticeHomepageQuery, GetApprenticeHomepageQueryResult>
+    {
+        private readonly IApprenticeAccountsApiClient<ApprenticeAccountsApiConfiguration> _accountsApiClient;
+        private readonly IApprenticeCommitmentsApiClient<ApprenticeCommitmentsApiConfiguration> _commitmentsApiClient;
+
```

Comment:
would this be better named _apprenticeCommitmentsApiClient or _apprenticeshipsApiClient. The name implies to me that we are calling the approvals commitments api. But it's up to you

Reply:
commitmentsApi is embedded in my head!